app/scripts/modules/core/src/pipeline/config/stages/preconfiguredJob/preconfiguredJobStage.ts

```diff
@@ -23,7 +23,9 @@ module(PRECONFIGUREDJOB_STAGE, []).run(() => {
           alias: 'preconfiguredJob',
           addAliasToConfig: true,
           restartable: true,
-          defaults,
+          defaults: {
+            parameters: paramDefaults,
```

Comment:
The values were previously under `defaults.parameters` but I missed this during the refactor